public/keymaps/uzu42_rev1_default.json

```
@@ -0,0 +1,31 @@
+{
+  "keyboard": "uzu42/rev1",
+  "keymap": "default_9b07098",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_Q",    "KC_W",   "KC_E",    "KC_R",    "KC_T",                                      "KC_Y",  "KC_U",            "KC_I",    "KC_O",    "KC_P",
+      "KC_A",    "KC_S",   "KC_D",    "KC_F",    "KC_G",                                      "KC_H",  "KC_J",            "KC_K",    "KC_L",    "KC_ENT",
+      "KC_Z",    "KC_X",   "KC_C",    "KC_V",    "KC_B",                                      "KC_N",  "KC_M",            "KC_COMM", "KC_DOT",  "KC_SLSH",
+      "KC_LSFT", "KC_SPC", "KC_LALT", "KC_LGUI", "MO(1)", "LCTL_T(KC_ESC)", "LSFT_T(KC_SPC)", "MO(2)", "RALT_T(KC_BSPC)", "KC_RCTL", "KC_RGUI", "KC_NO"
+    ],
+    [
+      "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",                          "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",
+      "KC_TAB",  "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",                         "KC_LEFT", "KC_DOWN", "KC_UP",   "KC_RGHT", "KC_ENT",
+      "KC_LCTL", "KC_SPC",  "KC_LSFT", "KC_LALT", "KC_DEL",                        "KC_BSPC", "KC_HOME", "KC_END",  "KC_NO",   "KC_NO",
+      "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "MO(3)",   "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"
+    ],
+    [
+      "KC_EXLM", "KC_AT",   "KC_HASH", "KC_DLR",  "KC_PERC",                       "KC_CIRC", "KC_AMPR", "KC_ASTR", "KC_LPRN", "KC_RPRN",
+      "KC_TAB",  "KC_UNDS", "KC_PLUS", "KC_PIPE", "KC_TILD",                       "KC_COLN", "KC_DQUO", "KC_GT",   "KC_LCBR", "KC_RCBR",
+      "KC_NO",   "KC_MINS", "KC_EQL",  "KC_BSLS", "KC_GRV",                        "KC_SCLN", "KC_QUOT", "KC_LT",   "KC_LBRC", "KC_RBRC",
+      "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "MO(3)",   "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"
+    ],
+    [
+      "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",                         "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",
+      "KC_F11",  "KC_F12",  "RESET",   "KC_PSCR", "KC_INS",                        "RGB_TOG", "RGB_HUI", "RGB_SAI", "RGB_VAI", "KC_LALT",

```

KC_LALT instead of KC_NO?

```suggestion
      "KC_F11",  "KC_F12",  "RESET",   "KC_PSCR", "KC_INS",                        "RGB_TOG", "RGB_HUI", "RGB_SAI", "RGB_VAI", "KC_NO",
```